crates/holochain_cascade/src/lib.rs

```diff
@@ -877,9 +867,9 @@ impl CascadeImpl {
         }
     }
 
-    #[cfg_attr(feature = "instrument", tracing::instrument(skip(self, options)))]
-    /// Gets an links from the cas or cache depending on it's metadata
+    /// Gets links from the DHT or cache depending on its metadata.
     // The default behavior is to skip deleted or replaced entries.
+    #[cfg_attr(feature = "instrument", tracing::instrument(skip(self, options)))]
```

Comment:
Really tempted to write a clippy lint for this, we have these both ways up all over the codebase

Reply:
At the very least we could open a clippy enhancement issue : )